batch/aiml-workloads/scripts/cleanup.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
```

Comment:
Is there any particular reason we're using `bash` instead of the more portable / common `/bin/sh`?

Reply:
No particular reason! Changed them to `sh`.